lib/oelite/baker.py

```diff
@@ -251,6 +254,7 @@ def show(self):
         #meta.dump(pretty=False, nohash=False, flags=True,
         #          ignore_flags=re.compile("filename|lineno"),
         meta.dump(pretty=True, show_nohash=(not self.options.nohash),
+                  dynvar_replacement=(not self.options.nodynvar),
```

Comment:
ugh more of this boolean arg stuff, it is really not making the code nicer to work with :)